appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Verifies the given attestation passes the given rego policy
+# usage:
+#   verify-attestation-with-policy.sh <attestation file> <output file> <passed file>
+# where:
+#   <attestation file> file containing the attestation to be verified
+#   <output file>      where to store the result in JSON format
+#   <passed file>      where to store the overall passing result
+set -euo pipefail
+
+attestation="$1"
+output="$2"
+passed="$3"
+
+predicate=$(jq '.payload | @base64d | fromjson' ${attestation})
+
+cd $(dirname $0)
+source lib/fetch.sh
+
+# TODO: If git clone fails, the task does not fail - fix that!
+./fetch-ec-policies.sh
+
+# TODO: fix the checks (other than not_useful) to handle the case when there
+# is no cluster access, i.e. no TaskRun yaml
+# When fixed, replace this line with a call to save-policy-config
+echo '{"non_blocking_checks":[
+    "not_useful",
+    "chains_config",
+    "cluster_sanity",
+    "transparency_urls",
+    "transparency_log_attestations"
+    ]}' | jq > $( json-data-file config policy )
+
+title Predicate
+# Show a shortened version of the predicate to avoid excessive logging
+echo -n $predicate | jq -r '.predicate = "..."'
+# Save the predicate for opa processing
+echo -n ${predicate} > $(json-data-file predicate)
```

Comment:
So the predicate *will* be visible to opa/rego under `data.predicate`, right?
This means we can't write a policy to validate the attestation _type or predicateType or the subject.
Not suggesting it's a problem, but just noting it. Is that intentional design choice? My instinct might be to throw in the full attestation in case we do want it visible in rego, but I'm not confident that's wise or correct.

(One of the (placeholder/demo) rego rules currently does look at the `_type` value at the top level of the attestation. We can easily ditch it though.)

Reply:
`_type` and `predicateType` are there. They are part of what gets decoded from the payload.

I think looking at these two fields from rego will be important for us.

Reply:
As per slack discussion, it makes sense to me now, thanks.

There's some potential for confusion around the terminogy, i.e. payload vs attestation vs predicate vs envelope and etc, but we'll figure it out.